prow/git/v2/client_factory.go

```diff
@@ -396,10 +396,11 @@ func (c *clientFactory) ClientForWithRepoOpts(org, repo string, repoOpts RepoOpt
 	// Initialize the new derivative repo (secondary clone) from the primary
 	// clone. This is a local clone operation.
 	timeBeforeSecondaryClone := time.Now()
-	if err := repoClientCloner.CloneWithRepoOpts(cacheDir, repoOpts); err != nil {
+	if err = repoClientCloner.CloneWithRepoOpts(cacheDir, repoOpts); err != nil {
```

Comment:
This commit is unnecessary because we already do a `return` if we fail, so we never observe the failure.

Reply:
Done. Changed the wrong place